content-src/components/StartupOverlay/StartupOverlay.jsx

```diff
@@ -14,8 +14,29 @@ export class _StartupOverlay extends React.PureComponent {
 
     this.state = {
       emailInput: "",
-      overlayRemoved: false
+      overlayRemoved: false,
+      flowId: "",
+      flowBeginTime: ""
```

Comment:
This was set to 0 in the original PR #4240 https://github.com/mozilla/activity-stream/pull/4240#discussion_r202423901